src/web/lib/getPrivacyFramework.ts

```diff
@@ -0,0 +1,23 @@
+import { getCountryCode } from '@frontend/web/lib/getCountryCode';
+
+const switches =
+    'guardian' in window &&
+    'config' in window.guardian &&
+    'switches' in window.guardian.config
+        ? window.guardian.config.switches
+        : {};
```

Comment:
I've noticed that changes made the the `window.guardian.config.switches` object seem persistent across page views, when testing locally. Might be worth double-checking how the config is set/read in DCR?